tests/contracts/test_normalization_of_return_types.py

```diff
@@ -28,7 +28,9 @@
                 '0xbb9bc244d798123fde783fcc1c72d3bb8c189413',
             ],
         ),
-    )
+    ),
+    ids=['nullbyte', 'soloaddr', 'addrlist']
```

Comment:
Assuming this is the fix for pytest 3.2, I think I'd rather just pin the dependency to `pytest<3.2.0` with a comment linking to the issue in the pytest repo.